bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/HtmlPageItem.java

```diff
@@ -0,0 +1,173 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.models;
+
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * This interface defines items that can be included on a page.
+ * @since com.adobe.cq.wcm.core.components.models 12.16.0
+ */
+public interface HtmlPageItem {
+
+    /**
+     * Property name that defines the type of the HTML element rendered by the page item
+     * @since com.adobe.cq.wcm.core.components.models 12.16.0
+     */
+    String PROP_ELEMENT = "element";
```

Comment:
we usually use `PN_` prefix for property name and `NN_` for node name.

Reply:
Updated in 9973de3